base-system/tools/installer/install.sh

```diff
@@ -83,9 +83,27 @@ while [ "$#" -gt 0 ]; do
 			exit 1
 		fi
 		;;
+	--instance-ip-address | -ia)
+		if [ -n "$2" ]; then
+			INSTANCE_IP_ADDRESS="$2"
+			shift 2
+		fi
+		;;
+	--instance-ip-mask | -im)
+		if [ -n "$2" ]; then
+			INSTANCE_IP_MASK="$2"
+			shift 2
+		fi
+		;;
+	--instance-ip-gateway | -ig)
+		if [ -n "$2" ]; then
+			INSTANCE_IP_GATEWAY="$2"
+			shift 2
+		fi
+		;;
 	-h | --help)
 		echo "Installs the current huronOS into an usb"
-		echo "Usage: ./install.sh [--root-password PASSWORD] [--directives-url URL] [--directives-server-ip IP]"
+		echo "Usage: ./install.sh [--root-password PASSWORD] [--directives-url URL] [--directives-server-ip IP] [--instance-ip-address|-ia IP --instance-ip-mask|-im NUMBER --instance-ip-gateway|-ig IP]"
```

Comment:
I would prefer that instead of `--instance-ip-address|-ia` you only set the large one even if internally it's possible the short one. 
Let's do another PR for setting a proper help page.

Reply:
Okay, on my way to make a PR on top of this one with help